build/webpack/webpack.config.js

```diff
@@ -0,0 +1,170 @@
+/*
+This is just porting all the configs from all the seperate transform files and
+having one large standard one. This should make it easier to update to webpack 2
+As well as just making it more accessible without having to dive around all the
+transforms. Would prefer to have a well commented file like Origami:
+*/
+
+const path =require('path');
+const glob = require('glob');
+const BowerWebpackPlugin = require('bower-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const autoprefixer = require('autoprefixer');
+
+
+const handlebarsConfig = () => {
+	const extraHelperDirs = glob.sync('**/node_modules/@financial-times/**/handlebars-helpers')
```

Comment:
This looks like some magic. You are finding all handlebars-helpers directories in all ft npm modules and adding them to your handlebars configuration. I'm assuming that this might cause issues if two handlebars helpers had the same function name, the last defined function overrides all the previous ones, right?